src/ext/components/partial-dependence-input.jade

```diff
@@ -11,20 +15,55 @@
       tr
         th
           label Model:
-        td
+        td(colspan='2')
           select(data-bind="options:models, value:selectedModel, optionsCaption:'(Select)'")
       tr
         th
           label Frame:
-        td
-          select(data-bind="options:frames, value:selectedFrame, optionsCaption:'(Select)'")
+        td(colspan='2')
+          select(data-bind="options:frames, value:selectedFrame, optionsCaption:'(Select)', event:{ change:updateColumns}")
```

Comment:
binds the change in selected frame to function that updates the list of columns. see the knockoutjs official docs for `data-bind`